app/assets/javascripts/components/locales/fr.jsx

```diff
@@ -33,7 +33,7 @@ const fr = {
   "tabs_bar.public": "Public",
   "tabs_bar.notifications": "Notifications",
   "compose_form.placeholder": "Qu’avez vous en tête&nbsp;?",
-  "compose_form.publish": "Publier",
+  "compose_form.publish": "Pouet",
```

Comment:
Yes!!! Pierrec [thought of it](https://mastodon.social/users/Pierrec/updates/100589) :)